draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -475,9 +475,160 @@ Additional JWT VC Issuer Metadata configuration parameters MAY also be used.
 
 ## JWT VC Issuer Metadata Validation
 
-The `issuer` value returned MUST be identical to the `iss` value of the JWT. If
-these values are not identical, the data contained in the response MUST NOT be
-used.
+The `issuer` value returned MUST be identical to the `iss` value of the
+JWT. If these values are not identical, the data contained in the response
+MUST NOT be used.
+
+# Type Metadata {#type-metadata}
+
+A type is associated with rules defining which claims may or must appear in the SD-JWT VC and whether they may, must, or must not be selectively disclosable.
+
+This specification defines Type Metadata that can be associated with a type of a SD-JWT VC as well as a method for retrieving the Type Metadata and processing rules. This Type Metadata is intended to be used, among other things, for the following purposes:
+
+ * Developers of Issuers and Verifiers can use the Typ Metadata to understand the
+   semantics of the type and the associated rules. While in some cases,
+   Issuers are the parties that define types (credential formats), this is
+   not always the case. For example, a type can be defined by a
+   standardization body or a community.
+ * Verifiers can use the metadata to determine whether a credential is valid
+   according to the rules of the type. For example, a Verifier can check
+   whether a credential contains all required claims and whether the claims
+   are selectively disclosable.
+ * Wallets can use the metadata to display the credential in a way that is
+   consistent with the Issuer's intent.
+
+Applications using this specification are called "consuming applications" in the following. This typically includes Issuers, Verifiers, and Wallets.
```

Comment:
We might also just call them "consumers".

Reply:
I like that better.